adapter/protobuf/convert.go

```diff
@@ -19,10 +19,37 @@ func ConvertValue(pv string, field entity.PrimitiveField) (interface{}, error) {
 		return nil, errors.New("type assertion failed")
 	}
 
+	protoDefaults := map[descriptor.FieldDescriptorProto_Type]interface{}{
```

Comment:
Can you initialize `protoDefaults` at the global scope?
It doesn't need to initialize many times.